schema.graphql

```diff
@@ -692,3 +692,40 @@ type Incentive @entity {
     rewardToken: Token!
     rewardPerDay: String!
 }
+
+# Nabla entities
+
+type Router @entity {
+    id: ID! # id can be the address of the router contract
+    swapPools: [SwapPool!] @derivedFrom(field: "router")
+    backstopPools: [BackstopPool!] @derivedFrom(field: "router")
+    paused: Boolean! # whether the router is paused
+}
+
+type NablaToken @entity {
```

Comment:
I had to rename from Token to NablaToken, since we already had a Token entity in the schema.

Reply:
Nice, we have to keep it in mind for the portal queries